src/geotop/math.optim.h

```diff
@@ -0,0 +1,10 @@
+#include <cmath>
```

Comment:
you forgot to put the guard!
```
#ifndef __math_optim_h_
#define __math_optim_h_
...

#endif 
```

Reply:
ok, I can add the guard but it still gives me the previous error if I use the not-templated ```power()``` function...